### PR TITLE
created initial html/css structure

### PR DESCRIPTION
**Pull request by matthewsearle01**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title></title>
```
Comment (BelindaWilliams) [inline index.html:6]: Enter title

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/styles.css">
+        <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
```
Comment (BelindaWilliams) [inline index.html:10]: Include defer

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                    <li>Zombies will pop up randomly</li>
+                    <li>Kill the zombies!</li>
+                    <li>Save the world</li>
+                    <li>Happy Halloween!</li>
+                </ol>
+            </div>
+            <div class="launch">
+                <a href="#">Launch Game</a>
+            </div>
+            <footer>&copy Agile Aardvarks 2021</footer>
```
Comment (BelindaWilliams) [inline index.html:31]: Need semicolon after &copy

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
+@import url('https://fonts.googleapis.com/css2?family=Creepster&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;500;600;700&display=swap');
+
```
Comment (BelindaWilliams) [inline css/styles.css:3]: Alphabetical order?

File: index.html (hunk 1 above)
Comment (matthewsearle01) [reply index.html:6]: Thanks, fixed!

**Pull request merged**